templates/vmset/cloudinit/userdata

```diff
@@ -12,4 +12,8 @@ chpasswd:
 {{- end }}
 bootcmd:
   - "mkdir /mnt/kubeconfig"
-  - "mount /dev/disk/by-id/virtio-fencingdisk /mnt/kubeconfig"
\ No newline at end of file
+  - "mkdir /root/kubeconfig"
```

Comment:
bootcmd will run on every boot. should be `mkdir -p` for later reboots, same for the above mkdir, or just use single `mkdir -p /mnt/kubeconfig /root/kubeconfig`